### PR TITLE
Fix a couple of errors descovered by coverity

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@mheon @baude @giuseppe @vrothberg @TomSweeneyRedHat PTAL

**Comment by rhatdan** [comment]
Found in:
https://bugzilla.redhat.com/show_bug.cgi?id=1721394

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Comment by mheon** [comment]
LGTM, but I'd like a nod from @giuseppe before merge

File: pkg/rootless/rootless_linux.c [hunk 1]
```diff
@@ -457,6 +457,12 @@ create_pause_process (const char *pause_pid_file_path, char **argv)
             }
 
           r = TEMP_FAILURE_RETRY (write (p[1], "0", 1));
+          if (r < 0)
+            {
+              fprintf (stderr, "cannot write to pipe: %s\n", strerror (errno));
+              kill (pid, SIGKILL);
```
Comment (giuseppe) [inline pkg/rootless/rootless_linux.c]: if we kill the `pid` process we would also need to cleanup the pid file.  Although if we get this far, it is probably safer to just let it run and avoid to kill the pause process.  Could you drop the `kill (pid, SIGKILL)` here?

Comment (rhatdan) [reply pkg/rootless/rootless_linux.c]: Removed,  PTAL

**Comment by giuseppe** [comment]
LGTM

**Pull request merged**